src/compiler/scala/tools/nsc/backend/jvm/CoreBTypes.scala

```diff
@@ -374,7 +374,7 @@ abstract class CoreBTypesFromSymbols[G <: Global] extends CoreBTypes {
     new Handle(Opcodes.H_INVOKESTATIC,
       coreBTypes.jliLambdaMetafactoryRef.internalName, sn.Metafactory.toString,
       MethodBType(
-        List(
+        Array[BType](
```

Comment:
Curious, is the type argument necessary?

Reply:
I think so because `apply` is overloaded and `Array` is invariant. I could actually remove the overload of `apply` now, it was really just a stepping stone for my refactoring.